build.rs

```diff
@@ -11,12 +19,17 @@ fn main() {
 
     vergen::vergen(config).expect("Unable to generate the cargo keys!");
 
-    // Download vue file
-    let vue_url = "https://unpkg.com/vue@3.0.5/dist/vue.global.js";
-    let mut resp = reqwest::blocking::get(vue_url)
-        .expect(&format!("Failed to download vue file: {}", &vue_url));
-    let vue_file_path = std::path::Path::new(env!("CARGO_MANIFEST_DIR")).join("src/html/vue.js");
-    let mut output_file = std::fs::File::create(&vue_file_path)
-        .expect(&format!("Failed to create vue file: {:?}", &vue_file_path));
-    std::io::copy(&mut resp, &mut output_file).expect("Failed to copy content.");
+    file_download(
+        "https://unpkg.com/vue@3.0.5/dist/vue.global.js",
+        "src/html/vue.js",
+    );
+    std::fs::create_dir_all("src/html/webrtc/adapter").unwrap();
+    file_download(
+        "https://webrtc.github.io/adapter/adapter-latest.js",
+        "src/html/webrtc/adapter/adapter-latest.js",
+    );
+    file_download(
+        "https://raw.githubusercontent.com/webrtcHacks/adapter/master/LICENSE.md",
```

Comment:
avoid using master, please use a specific hash to avoid problems or license change and etc. We should use the same hash of the current adapter.
`https://github.com/webrtcHacks/adapter/blob/b725bee/LICENSE.md`

Reply:
We should also avoid using adapter-latest from master, but since this is used by the internet without a specific version, we should be fine.

Reply:
> 

It is the recommended version, so I think we should try to keep up with that.